### PR TITLE
Fix broken new enumeration links

### PR DESCRIPTION
**Pull request by oliverguenther**
New enumerations path was / is broken in 5.0.0

Related to https://github.com/opf/openproject-documents/pull/48

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://oliverguenther-fix-enumeration-links.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://oliverguenther-fix-enumeration-links.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: app/views/enumerations/index.html.erb [hunk 1]
```diff
@@ -103,7 +103,9 @@ See doc/COPYRIGHT.rdoc for more details.
     </div>
   <% end %>
   <div class="generic-table--action-buttons">
-    <%= link_to l(:label_enumeration_new), { action: 'new', type: klass.name , class: 'button -alt-highlight'} do %>
+    <%= link_to({ action: 'new', type: klass.name },
```
Comment (ulferts) [inline app/views/enumerations/index.html.erb]: why not use the named paths like you did in the spec: `new_enumeration_path(type: klass.name)`

**Comment by ulferts** [comment]
Free to merge on Travis' agreement.

**Comment by oliverguenther** [comment]
I even addressed your remark. Totally self merging now.

**Pull request merged**